diff --git a/node_modules/@react-native-firebase/analytics/android/build.gradle b/node_modules/@react-native-firebase/analytics/android/build.gradle
index 0165f28..e32a886 100644
--- a/node_modules/@react-native-firebase/analytics/android/build.gradle
+++ b/node_modules/@react-native-firebase/analytics/android/build.gradle
@@ -105,6 +105,7 @@ if (rootProject.ext && rootProject.ext.firebaseJson) {
 }
 
 android {
+  namespace 'io.invertase.firebase.analytics'
   defaultConfig {
     multiDexEnabled true
     manifestPlaceholders = [
