diff --git a/node_modules/@react-native-firebase/app/android/build.gradle b/node_modules/@react-native-firebase/app/android/build.gradle
index feb152f..65817db 100644
--- a/node_modules/@react-native-firebase/app/android/build.gradle
+++ b/node_modules/@react-native-firebase/app/android/build.gradle
@@ -59,6 +59,7 @@ if (rootProject.ext && rootProject.ext.firebaseJson) {
 }
 
 android {
+  namespace 'io.invertase.firebase'
   defaultConfig {
     multiDexEnabled true
     manifestPlaceholders = [
